docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/install-on-kubernetes.md

```diff
@@ -14,8 +14,20 @@ three-node cluster is created. You use a Helm chart our support team provides to
 
 To get started with Palette on Kubernetes, refer to the [Install Instructions](install.md) guide.
 
+## Get Started
+
+Select the scenario and the corresponding guide to install Palette on Kubernetes. If you are installing Palette in an
+refer to the environment preparation guide before installing Palette.
+
+| Scenario                                                 | Environment Preparation Guide                                                | Install Guide                             |
+| -------------------------------------------------------- | ---------------------------------------------------------------------------- | ----------------------------------------- |
+| Install Palette on Kubernetes with internet connectivity | None                                                                         | [Install Instructions](install.md)        |
+| Install Palette on Kubernetes in an airgap environment   | [Kubernetes Airgap Preperation](../airgap/kubernetes-airgap-instructions.md) | [Airgap Install Instructions](install.md) |
```

Comment:
Airgap Install Instructions md file seems to be incorrect

Reply:
@shubhamrajvanshi not sure I am following. What do you mean?

Reply:
fixed